packages/xsnap/lib/text-shim.js

```diff
@@ -7,14 +7,21 @@ const { fromString } = ArrayBuffer;
 const { fromArrayBuffer } = String;
 
 class TextEncoder {
-  encode(s) {
-    return new Uint8Array(fromString(s));
+  encode(text) {
+    return new Uint8Array(fromString(text));
   }
 }
 
 class TextDecoder {
-  decode(bs) {
-    return fromArrayBuffer(bs);
+  decode(bytes) {
+    // TODO: the following condition can be removed in a future update of XS.
```

Comment:
I suppose it's OK if this TODO never gets done, so this isn't critical, but...

What are the norms around TODOs around here? Should this have a pointer to an issue? In the moddable repo? Or should it be down-graded from TODO to NOTE before merging or something?

Reply:
Added a link. I don’t know the norms, but I think tracking with an issue should suffice. Thanks for the suggestion.